### ISSUE TITLE
[App] GitLab

### ISSUE THREAD

**Issue by alexellis** [alexellis/arkade#163]
## Expected Behaviour

Offer `gitlab` as an app using its helm chart:

* https://docs.gitlab.com/charts/
* https://docs.gitlab.com/charts/installation/

## Context

From the docs:

> Note: There are currently known limitations when using this chart, and not all features of GitLab are available.

Suggested on Twitter -> https://twitter.com/alexellisuk/status/1288518379607937024

**Comment by Johannestegner** [comment]
Hi there!

Started writing on this in a fork (https://github.com/Johannestegner/arkade) (branch `feature/gitlab-app` which is not yet pushed).  
This will be my first contribution to this repository and GitLab is quite a huge application (with many services), so I'm thinking to open up the following optional flags:

```
--ce (Install the Community Edition of GitLab (default: false))
--postgresql (Install PostgreSQL alongside GitLab (default: false))
--redis (Install Redis alongside GitLab (default: false))
--minio (Install MinIO alongside GitLab (default: false))
```

I feel that the default values of persistent storage and other things like these are quite sane and I can't imagine someone wants to install the suite without any storage at all, and if they do, they can use `--set` to do so.

I also consider if the optional additions should even be possible to choose in this installation (that is Prometheus, Grafana and GitLab runner).

Here is a link to all the global variables available:

https://docs.gitlab.com/charts/charts/globals.html

Does this sound like a good idea?

**Comment by Johannestegner** [comment]
Correction to this:

> ```
> --ce (Install the Community Edition of GitLab (default: false))
> --postgresql (Install PostgreSQL alongside GitLab (default: false))
> --redis (Install Redis alongside GitLab (default: false))
> --minio (Install MinIO alongside GitLab (default: false))
> ```



```
 --ce (Install the Community Edition of GitLab (default: false))
 --postgresql (Install PostgreSQL alongside GitLab (default: true))
 --redis (Install Redis alongside GitLab (default: true))
 --minio (Install MinIO alongside GitLab (default: true))
```

**Comment by Waterdrips** [comment]
https://github.com/alexellis/arkade/pull/252

**Comment by Waterdrips** [comment]
/close
implemented in #252 Thanks
